dirty_cat/datasets/tests/test_fetching.py

```diff
@@ -1,111 +1,106 @@
 from tempfile import TemporaryDirectory
-from typing import Callable, Tuple
 from unittest import mock
 from urllib.error import URLError
 
 import pandas as pd
 import pytest
+from openml.datasets import check_datasets_active
```

Comment:
Should we do an optional import and optional test so that the lone developer can run tests on her laptop without having this packaged installed.